docs/source/en/api/attnprocessor.mdx

```diff
@@ -0,0 +1,39 @@
+# Attention Processor
+
+An attention processor is a class for applying different types of attention mechanisms.
+
+## AttnProcessor
+[[autodoc]] models.attention_processor.AttnProcessor
+
+## AttnProcessor2_0
+[[autodoc]] models.attention_processor.AttnProcessor2_0
+
+## LoRAAttnProcessor
+[[autodoc]] models.attention_processor.LoRAAttnProcessor
+
+## CustomDiffusionAttnProcessor
+[[autodoc]] models.attention_processor.CustomDiffusionAttnProcessor
+
+## AttnAddedKVProcessor
```

Comment:
I couldn't really find anything about what `AddedKV` attention is so I'd love some help here :) cc @sayakpaul @patrickvonplaten @yiyixuxu @pcuenca

Reply:
`AttnAddedKVProcessor` is the regular `AttnProcessor` but with extra learnable matrices for keys and values. These are used on the encoder hidden states corresponding to the text encoder.